static/js/biomedical/disease/drug_table.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import { GRAPH_BROWSER_REDIRECT } from "../bio_charts_utils";
+import { CompoundDiseaseTreatmentData } from "./chart";
+export interface DrugTableProps {
+  data: CompoundDiseaseTreatmentData[];
+}
+export function DrugTable(props: DrugTableProps): JSX.Element {
+  let DrugTableData = props.data;
+  // takes the top 10 chemical compound disease associations for greater than 10 values
+  if (DrugTableData.length > 10) {
+    DrugTableData = DrugTableData.slice(0, 10);
+  }
+  // sorts the array based on FDA clinical phase number
+  DrugTableData = DrugTableData.sort(
+    (a, b) => b.clinicalPhaseNumber - a.clinicalPhaseNumber
+  );
+  return (
+    <table>
+      <thead>
+        <tr>
+          <td>Parent Node</td>
+          <td>Compound ID</td>
+          <td>Compound Name</td>
+          <td>FDA Clinical Phase</td>
+        </tr>
+      </thead>
+      <tbody>
+        {DrugTableData.map((item, idx) => {
+          return (
+            <tr>
+              <td key={idx}>
```

Comment:
you can actually just put the key in the <tr> tag & don't need the key for each of the <td>s

Reply:
I'm sorry, I don't think I quite understand this. do you mean something like `<td = {idx + key}...></td>` because then key won't be defined inside the tag

Reply:
I mean you can have `<tr key={idx}>` and then you don't have to set the key for `<td>`s

Reply:
Got it! Thank you, implemented your suggestion!